### PR TITLE
Submit to v1 and v2

### PR DESCRIPTION
**Pull request by rail**
This simplifies the workflow and dosn't require any in-tree changes.

### CODE REVIEW

**Comment by rail** [comment]
This can be deployed only after we switch Thunderbird to shipit v2

File: shipitscript/script.py [hunk 1]
```diff
@@ -33,17 +33,12 @@ def mark_as_shipped_action(context):
     release_name = context.task['payload']['release_name']
 
     log.info('Marking the release as shipped ...')
-    ship_actions.mark_as_shipped(context.ship_it_instance_config,
```
Comment (MihaiTabara) [inline shipitscript/script.py]: 🔥

**Comment by rail** [comment]
Staging worked fine: https://tools.taskcluster.net/groups/CuxxskuuRa6e611utCF0NQ/tasks/XynTlj_tQsWF0ptwoXusXg/runs/5

**Comment by rail** [comment]
time to update the nix files \o/

**Comment by rail** [comment]
It also triggered this product details update: https://github.com/mozilla-releng/product-details/commit/9d9aac0727840f6fc7396faa9f1996be4434d048

**Pull request merged**